content/en/quality_gates/_index.md

```diff
@@ -106,18 +96,18 @@ To create a Quality Gate rule, see the [Setup documentation][2].
 
 You can evaluate and update quality control processes by accessing Quality Gates rules on the [**Quality Gates Rules** page][6]. Improve your deployment practices based on your project requirements and desired performance outcomes. 
 
-{{< img src="quality_gates/rules_list_1.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
+{{< img src="quality_gates/rules_list_2.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
```

Comment:
```suggestion
{{< img src="quality_gates/setup/rules_list_2.png" alt="List of Quality Gate rules in Datadog" style="width:100%" >}}
```

Reply:
The image link is broken. I believe this is the correct path to the image.

Reply:
Ah oops, I put the image file in the wrong place! I'm going to reject this suggestion and instead move the file to where I originally meant to put it